docs/guide-2.x.md

```diff
@@ -1,9 +1,180 @@
+    # for more information.
+    include /etc/nginx/conf.d/*.conf;
+
+    server {
+        listen       80 default_server;
+        listen       [::]:80 default_server;
+        server_name  _;
+        root         /usr/share/ercole/web;
+
+        # Load configuration files for the default server block.
+        include /etc/nginx/default.d/*.conf;
+
+        location /data/ {
+            proxy_pass http://127.0.0.1:11111/;
+        }
+
+        location /api/ {
+            proxy_pass http://127.0.0.1:11113/;
+        }
+
+        location /repo/ {
+            proxy_pass http://127.0.0.1:11114/;
+        }
+
+        location /chart/ {
+            proxy_pass http://127.0.0.1:11116/;
+        }
+
+
+        error_page 404 /404.html;
+            location = /40x.html {
+        }
+
+        error_page 500 502 503 504 /50x.html;
+            location = /50x.html {
+        }
+    }
+
+
+    # Settings for a TLS enabled server.
+    #
+    server {
+        listen       443 ssl http2 default_server;
+        listen       [::]:443 ssl http2 default_server;
+        server_name  _;
+
+        ssl_certificate "/etc/pki/nginx/server.crt";
+        ssl_certificate_key "/etc/pki/nginx/private/server.key";
+        ssl_session_cache shared:SSL:1m;
+        ssl_session_timeout  10m;
+        ssl_ciphers PROFILE=SYSTEM;
+        ssl_prefer_server_ciphers on;
+
+        root         /usr/share/ercole/web;
+
+        # Load configuration files for the default server block.
+        include /etc/nginx/default.d/*.conf;
+
+        location /data/ {
+            proxy_pass http://127.0.0.1:11111/;
+        }
+
+        location /api/ {
+            proxy_pass http://127.0.0.1:11113/;
+        }
+
+        location /repo/ {
+            proxy_pass http://127.0.0.1:11114/;
+        }
 
+        location /chart/ {
+            proxy_pass http://127.0.0.1:11116/;
+        }
+
+        error_page 404 /404.html;
+            location = /40x.html {
+        }
+
+        error_page 500 502 503 504 /50x.html;
+            location = /50x.html {
+       }
+    }
+}
+```
+* `mkdir -p /etc/pki/nginx/private`
+* `openssl req -nodes -x509 -newkey rsa:4096 -keyout /etc/pki/nginx/private/server.key -out /etc/pki/nginx/server.crt`.
+* Add the following lines to /etc/ercole/conf.d/20-ercolesetup.toml:
+```
+# FE.APIService.RemoteEndpoint = "https://<myip>/api"
+# FE.ChartService.RemoteEndpoint = "https://<myip>/chart"
+```
+* If your machine has multiple IP addresses, check and fix endpoints in `/etc/ercole/conf.d/20-ercolesetup.toml`.
+* If you want change the URI used to connect to mongodb, run the command `echo 'Mongodb.URI = "mongodb://user:pass@ip:port/ercole"' > /etc/ercole/conf.d/40-mongoconf.toml` with the right [Mongo URI](https://docs.mongodb.com/manual/reference/connection-string/).
+* Review ercole configuration with `ercole show-config`
+* If you have changed ercole configuration, run the command `ercole-setup`
```

Comment:
I don't hink it's needed to run `ercole-setup` each time a configuration file has been changed! :thinking:

Reply:
It's needed because it's launch ercoleweb-setup that update ercole-web config.json (/usr/share/ercole/web/config.json) and remigrate the database

Reply:
:+1: